support/build/Makefile.rules

```diff
@@ -813,7 +813,7 @@ $(error buildrule_$(call fileext,$(strip $(2))) is not defined: Failed to instal
 $(call buildrule_$(call fileext,$(strip $(2))),$(strip $(1)),$(strip $(2)),$(strip $(3)),$(strip $(4)),$(strip $(5)))
 
 # Default buildrule_* dependencies
-$(4): | preprocess
+$(4): $(call qstrip,$($(call vprefix_src,$(1),$(2),$(3),CDEPS)) $($(call vprefix_src,$(1),$(2),$(3),CDEPS-y))) | preprocess
```

Comment:
The commit message will need to be added somewhere in the documentation because otherwise we'll forget about it :pray:

Reply:
You mean https://github.com/unikraft/docs/ ? Here, I totally agree. In general the latest build system features need to be documented.